code-challenges.test.js

```diff
@@ -45,23 +82,43 @@ const studyMinutesWeek2 = {
   wednesday: 60,
   thursday: 20,
   friday: 15,
-  saturday: 65
-}
+  saturday: 65,
+};
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
-
+const testArray = (smw) => {
+  return Object.values(smw).sort();
+};
+// so the test passes with the first parameter but not the second one. i'm not sure why. moving to the next question.
```

Comment:
Just a reminder to paste any errors you are getting when running the test in here.  Your logic on this problem looks good, however, sort by itself on numbers behaves differently than expected.  We have to provide more information to it so that it will sort the numbers in ascending or descending order.